### PR TITLE
align UI style more closely to figma designs

### PR DESCRIPTION
**Pull request by alexbowen**
this updates task to include resolutions to a few misunderstandings about figma designs
including the UI to hide/show the filter sidebar with was removed in subsequent PR. the search filters styling is left in component for use when this arises

https://dfedigital.atlassian.net/browse/TEVA-1123

Screenshot 2020-08-12 at 08 40 05
Screenshot 2020-08-12 at 08 39 54

### CODE REVIEW

File: app/components/hiring_staff/vacancies_component.html.haml [hunk 1]
```diff
@@ -1,12 +1,19 @@
...
 
-      = f.hidden_field :jobs_type, value: @selected_type
+        = f.hidden_field :jobs_type, value: @selected_type
 
-%div{ class: grid_column_class }
+.moj-filter-layout__content{ class: grid_column_class }
+  .dashboard-header{ class: "govuk-!-margin-bottom-5" }
+    - if @organisation.is_a?(SchoolGroup)
+      .filters-information.moj-action-bar
+        %button.govuk-button.govuk-button--secondary{ class: "govuk-!-margin-bottom-0 moj-action-bar__filter", id: "toggle-filters-sidebar" }= t('buttons.hide_filters')
```
Comment (cpjmcquillan) [inline app/components/hiring_staff/vacancies_component.html.haml:14]: We can add `.moj-action-bar__filter#toggle-filters-sidebar` if you prefer. We don't have a style guide for haml or any linting at the minute so it doesn't really matter though

File: app/frontend/src/lib/ui/components/filterGroup.scss [hunk 2]
```diff
@@ -1,18 +1,41 @@
-.moj-filter {
+.filter-group__container {
+  border-bottom: 1px solid #b1b4b6;
```
Comment (cpjmcquillan) [inline app/frontend/src/lib/ui/components/filterGroup.scss:2]: Is this a govuk-colour?

File: app/frontend/src/lib/ui/components/filterGroup.scss [hunk 3]
```diff
@@ -1,18 +1,41 @@
...
-    font-size: 1rem;
+  .moj-filter__header {
+
+    padding: 8px govuk-spacing(2);
+
+    &.search-filters {
+      background: none;
+      border: 1px solid govuk-colour("white");
+      padding: 0;
+      border-bottom: 2px solid #b1b4b6;
```
Comment (cpjmcquillan) [inline app/frontend/src/lib/ui/components/filterGroup.scss:35]: Same with this colour

File: app/components/hiring_staff/vacancies_component.html.haml (hunk 1 above)
Comment (alexbowen) [reply app/components/hiring_staff/vacancies_component.html.haml:14]: lets wait till haml linting, there a few things like " or ' too which is different in many places

File: app/frontend/src/lib/ui/components/filterGroup.scss (hunk 2 above)
Comment (alexbowen) [reply app/frontend/src/lib/ui/components/filterGroup.scss:2]: no it isnt a gov color, i used the color that is the border for the tabs/table

**Pull request merged**